### ISSUE TITLE
Icon rendering issue in terminal prompt when GNOME res scaling is not 100%.

### ISSUE THREAD

**Issue by mogwaidao** [ublue-os/bluefin#1201]
### Describe the bug

I am on bluefin-dx:latest, with F40.

When GNOME resolution scaling is set to anything other than 100%, it causes the container/package icon in the default CLI prompt in Ptyxis to render as the wrong size (in my case - as a very large icon partially obscuring text output on the left hand size of the terminal). This also sometimes cause rendering anomaly in Ptyxis (for me, it presents as flashing partial icon rendered in the lower left corner within the terminal).

I have also confirmed this issue on two different machines on bluefin-dx:latest.

### What did you expect to happen?

Regardless of the GNOME resolution scaling setting, the icon renders in relative same size, when compared to the font. Icon should not obscure terminal output.

### Output of `rpm-ostree status`

```shell
❯ rpm-ostree status
State: idle
AutomaticUpdates: stage; rpm-ostreed-automatic.timer: no runs since boot
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:latest
                   Digest: sha256:e97161aef35b1beacb613da970a8383c4af107357ebf738b5421f274da55e05b
                  Version: 40.20240423.0 (2024-04-24T14:53:09Z)

  ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:latest
                   Digest: sha256:fad2be99a5da85fa011a163594749e96457c9c16abbe63b8a351ae3b458dc763
                  Version: 39.20240422.0 (2024-04-22T20:35:24Z)
```

### Extra information or context

This is likely not a Bluefin bug per se. However, since the ctrl-alt-* container profiles are pre-configured and shipped as part of latest stable, this issue is likely to affect many Bluefin users and causing a lot of head scratching, since the cause may  not be obvious, and that this issue does prevent the CLI fully display potentially useful output.

It maybe prudent to adjust the pre-configured default prompt to not include this icon, until the underlying cause is addressed.

For those encountering this issue with HiDPI displays, the non-ideal but functional workaround is to set your display resolution to the scaled effective resolution and set the scaling factor back to 100% at the new effective resolution. The icon will render correctly in the terminal (100% scaling factor).

For example, instead of scaling 150% on a 3840x2160 resolution display to make text readable, set resolution to 2560x1440 scaled to  100% (same effective resolution) instead. The prompt CLI icon should now render as intended in the terminal.

**Comment by castrojo** [comment]
Someone reported this in chat and we rebuilt ptyxis, can you `ujust update` and reboot to confirm? We think we got it.

**Comment by mogwaidao** [comment]
> Someone reported this in chat and we rebuilt ptyxis, can you `ujust update` and reboot to confirm? We think we got it.

Performed above. Still seeing issue.

❯ rpm-ostree status
State: idle
AutomaticUpdates: stage; rpm-ostreed-automatic.timer: no runs since boot
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:latest
                   Digest: sha256:e97161aef35b1beacb613da970a8383c4af107357ebf738b5421f274da55e05b
                  Version: 40.20240423.0 (2024-04-24T14:53:09Z)

  ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:latest
                   Digest: sha256:fad2be99a5da85fa011a163594749e96457c9c16abbe63b8a351ae3b458dc763
                  Version: 39.20240422.0 (2024-04-22T20:35:24Z)

![Screenshot from 2024-04-24 18-55-38](https://github.com/ublue-os/bluefin/assets/23100517/10701109-5120-4021-909e-96e3d3d97152)

**Comment by castrojo** [comment]
Looks like we only solved it for some folks, we'll keep investigating, thanks!

**Comment by lauretano** [comment]
tried the gnome nightly flatpak ptyxis, since I use it on an Asahi Linux on Wayland with fractional scaling, and don't run into the emoji font scaling issue there. 

![image](https://github.com/ublue-os/bluefin/assets/1905121/115cb1cc-7274-4367-8453-708e0f5fe922)

**Comment by m2Giles** [comment]
We just removed a package that was blocking F40 from building for Bluefin. On an image newer than 24 April this should be resolved (at least on my machines Aurora-DX/Bluefin-DX 40)

**Comment by RealVishy** [comment]
i can confirm that this issue is resolved on aurora with my image.

Version: 40.20240427.0 (2024-04-27T23:13:59Z)

**Comment by mogwaidao** [comment]
I just confirmed that the issue is resolved here on dx:latest (Version: 40.20240427.0 (2024-04-27T23:14:34Z).
